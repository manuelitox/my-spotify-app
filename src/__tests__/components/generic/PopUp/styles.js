import React from 'react'
import { shallow } from 'enzyme'

import StyledPopUp from 'components/generic/PopUp/styles'

describe('Component | StyledPopUp', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledPopUp />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})