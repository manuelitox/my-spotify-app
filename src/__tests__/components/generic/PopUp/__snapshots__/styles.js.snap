// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledPopUp should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-color: white;
  height: 100vh;
  top: 0;
  overflow: auto;
  padding: 20px;
  position: absolute;
  right: 0;
  width: ",
          [Function],
          ";
  z-index: 5;
  @media only screen and (max-width: ",
          [Function],
          ") {
    width: 100%;
  }
",
        ],
      },
      "defaultProps": Object {
        "width": "500px",
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "width": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  width="500px"
/>
`;
